pkg/serverconfig/config.go

```diff
@@ -178,4 +184,21 @@ func addCustomization(fs *flag.FlagSet, customization *Customization) {
 	if customization.CustomLogoFile != "" {
 		fs.Set("custom-logo-file", customization.CustomLogoFile)
 	}
+
+	categories, err := json.Marshal(&customization.DeveloperCatalog.Categories)
+	if err == nil {
+		fs.Set("developer-catalog-categories", string(categories))
+	} else {
+		fmt.Printf("Could not marshal ConsoleConfig customization.developerCatalog.categories: %v", err)
```

Comment:
This should be a fatal error.

Reply:
Changed to klog.Fatalf instead of fmt.Printf